app/views/smart_proxies/logs/_failed_modules.html.erb

```diff
@@ -0,0 +1,3 @@
+<% modules.each do |module_name, error_message| %>
+  <a href="#logs" onclick="filterLogsByMessage('<%= module_name %>'); setTab();" data-toggle="tooltip" data-placement="top" title="<%= error_message %>"><%= module_name.capitalize %></a>
```

Comment:
You can use `Feature.name_map[module_name]` instead of `module_name.capitalize` to get the correct feature name capitalization.